hap-ibd/dockerfile

```diff
@@ -0,0 +1,50 @@
+################## BASE IMAGE ######################
+FROM adoptopenjdk:8-jre@sha256:bf736024c65942ab045c497e408706f9e599651bf8c01a75ecb3d6971d66700d
+
+################## METADATA ######################
+LABEL base_image="biocontainers:vdebian-buster-backports_cv1"
+LABEL version="1"
+LABEL software="hap-ibd"
+LABEL software.version="1.0, 20May22.818"
+LABEL about.summary="Detects identity-by-descent (IBD) segments and homozygosity-by-descent (HBD) segments in phased genotype data"
+LABEL about.home="https://github.com/browning-lab/hap-ibd"
+LABEL about.documentation="https://github.com/browning-lab/hap-ibd/blob/master/README.md"
+LABEL about.publication="Y Zhou, S R Browning, B L Browning. A fast and simple method for detecting identity by descent segments in large-scale data. The American Journal of Human Genetics 106(4):426-437. doi: https://doi.org/10.1016/j.ajhg.2020.02.010"
+LABEL about.license_file="https://github.com/browning-lab/hap-ibd/blob/master/LICENSE-2.0.txt"
+LABEL about.license="Apache-2.0"
+LABEL extra.identifiers.biotools="hap-ibd"
+LABEL about.tags="Genomics"
+
+################## MAINTAINER ######################
+MAINTAINER Ashot Margaryan <ashotmarg2004@gmail.com>
+
+################## INSTALL wget ####################
+RUN  apt-get update \
+  && apt-get install -y wget \
+  && rm -rf /var/lib/apt/lists/*
+
+################## CREATE USER #####################
+RUN groupadd -g 901 biodocker \
```

Comment:
Not needed if using biocontainers base image

Reply:
Sure, will remove this part :)